src/adafruit_blinka/microcontroller/fake_mcp2221/i2c.py

```diff
@@ -0,0 +1,29 @@
+# SPDX-FileCopyrightText: 2021 Melissa LeBlanc-Williams for Adafruit Industries
+#
+# SPDX-License-Identifier: MIT
+"""I2C Class for MCP2221"""
+import random
+from .fake_mcp2221 import mcp2221
+
+
+class I2C:
+    """Custom I2C Class for MCP2221"""
+
+    def __init__(self, *, frequency=100000):
+        self._mcp2221 = mcp2221
+        self._freq = frequency
+
+    def scan(address_list=None):
```

Comment:
You might want to add the @staticmethod decorator here

Reply:
@makermelissa Added!